src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlAuthenticationProviderManager.NetCoreApp.cs

```diff
@@ -59,6 +59,24 @@ public SqlAuthenticationProviderManager(SqlAuthenticationProviderConfigurationSe
                 return;
             }
 
+            if (!string.IsNullOrEmpty(configSection.ApplicationClientId))
+            {
+                try
+                {
+                    _applicationClientId = configSection.ApplicationClientId;
+                }
+                catch (Exception e)
```

Comment:
when would this ever throw on line 66 but not throw on line 62? why do you need a try/catch at all?

Reply:
Yes that's right.
I believe in other cases, we have type conversions, hence the try/catch, but not needed here.